scenes/enemies/MoldedDruvar/MoldedDruvar.gd

```diff
@@ -50,7 +50,8 @@ func _ready():
 
 
 func _add_loot():
-	add_item_to_loottable("Gold", 25, 100)
+	add_item_to_loottable("Gold", 0.75, 300)
+	add_item_to_loottable("Club", 0.05, 1)
```

Comment:
Gonna sound silly, but thematically speaking, should it be an axe that has a chance to drop?

Reply:
Oh wait, the Axe is like god-tier strong in comparison, nevermind.

Reply:
also the loot isn't really sorted out in general